desktop/cmp/dataview/DataView.js

```diff
@@ -29,50 +28,20 @@ export class DataView extends Component {
 
     baseClassName = 'xh-data-view';
 
-    constructor(props) {
-        super(props);
-        const {store, selModel, contextMenuFn, itemRenderer, emptyText} = props.model;
-        this._gridModel = new GridModel({
-            store,
-            selModel,
-            contextMenuFn,
-            emptyText,
-            columns: [
-                {
-                    colId: 'data',
-                    flex: true,
-                    elementRenderer: itemRenderer,
-                    agOptions: {
-                        valueGetter: this.valueGetter
-                    }
-                }
-            ]
-        });
-    }
```

Comment:
This looks to me like a clear improvement - funny that we had this hanging on the component for so long, when we've always had a dedicated model class....